src/core/analytics.js

```diff
@@ -742,6 +766,24 @@ class Analytics {
       // convert integrations object to server identified names, kind of hack now!
       transformToServerNames(rudderElement.message.integrations);
 
```

Comment:
Add same comment of example objects & expected merge object

Reply:
[ref](https://github.com/rudderlabs/rudder-sdk-js/pull/805#discussion_r1055137246)